src/core/mac/link_raw.hpp

```diff
@@ -237,13 +242,31 @@ class LinkRaw
      */
     otError SetExtAddress(const otExtAddress &aExtAddress);
 
+    /**
+     * This method gets the transmit frame.
+     *
+     * @returns A pointer to the transmit frame.
+     *
+     */
+    otRadioFrame *GetTransmitFrame(void) { return mTransmitFrame; }
+
 private:
-    otInstance &mInstance;
+    enum
+    {
+        kOperationTransmitData = 1 << 0,
+    };
+
+    void        TransmitNow(void);
+    void        StartTransmit(void);
+    static void HandleOperationTask(Tasklet &aTasklet);
+    void        HandleOperationTask();
+
+    Tasklet  mOperationTask;
+    uint16_t mPendingOperations;
```

Comment:
Are there plans to add more pending operations? If not, declare this as type bool and 1-bit bit field?

Reply:
I thought this class could be finally be base class of Mac. Declaring this way could share this variable with its subclass. I have modified according to your suggestion.